packages/demo-site/scripts/start-dev.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const { spawn, execSync } = require('child_process')
+const path = require('path')
+
+const templateFolder = path.resolve(__dirname, '../tpls/')
+
+const buildScript = 'yarn cross-env NODE_ENV=DEV yarn build:prod'
+const consolePrefix = '[demo-site]:'
+
+fs.watch(templateFolder, () => {
```

Comment:
I recommend using [chokidar](https://www.npmjs.com/package/chokidar) because it's better in terms of performance than `fs.watch`

Reply:
@vuhuucuong That's cool. Updated code to use chokidar